src/main/java/com/shippingcostestimator/enterprise/dao/ShipmentDAOStub.java

```diff
@@ -38,7 +38,7 @@ public Shipment saveEstimate(Shipment shipment) {
 
     /***
      * Returns a list of Shipment objects that are currently in use by the program.
-     * @return
+     * @return A list of shipments.
      */
```

Comment:
Added return description JavaDoc. [Ref](https://medium.com/@rhamedy/a-short-summary-of-java-coding-best-practices-31283d0167d3)